backend/Recodo.Common/Enums/NotificationType.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recodo.Common.Enums
+{
+    public enum NotificationType
+    {
+        BillingNotification,
+        NewVideoNotification,
+        NewReactionNotification,
+        NewComment
```

Comment:
```suggestion
        NewVideo,
        NewReaction,
        NewComment
```

Remove redundant type and keep naming consistency